Automattic/wp-calypso
# Gutenboarding: remove site type

#### Changes proposed in this Pull Request

* Removes site type question in Gutenboarding:

##### Before

<img width="965" alt="Screenshot 2019-11-21 at 11 23 37" src="https://user-images.githubusercontent.com/87168/69324584-671c1780-0c51-11ea-93fa-95edcebe48d7.png">

##### After

![image](https://user-images.githubusercontent.com/87168/69325467-e1996700-0c52-11ea-959e-f1edd854710a.png)


#### Testing instructions

- Open `/gutenboarding` and see the first question gone ✨ 
- Confirm that there is no code related to `SiteType` left around
